app/routes/$lang.about.tsx

```diff
@@ -2,20 +2,35 @@ import { Layout } from "~/components/Layout"
 import { Typography } from "~/components/Typography"
 import isLangSupportedLang from "~/sanity/queries/isLangSupportedLang"
 import { json, LoaderFunction } from "@remix-run/node"
+import { getStaticPageByRoute, StaticPage } from "~/sanity/queries/staticPages"
+import { client } from "~/sanity/client"
+import { useLoaderData } from "@remix-run/react"
+import { PortableText } from "@portabletext/react"
+import PortableTextComponents from "~/components/PortableTextComponents"
+import { useOtherLanguage } from "~/lib/useOtherLanguage"
+
+type StaticPageLoaderData = {
+  staticPage: StaticPage
+}
 
 export const loader: LoaderFunction = async ({ params }) => {
   isLangSupportedLang(params.lang)
 
-  return json({}, { status: 200 })
+  const staticPage = await getStaticPageByRoute(client, params.lang, "/about")
+
+  return json({ staticPage }, { status: 200 })
 }
 
 const Privacy = () => {
+  const { staticPage } = useLoaderData() as StaticPageLoaderData
+
   return (
-    <Layout useMargins>
-      <Typography.H1>About Us</Typography.H1>
-      <Typography.TextSmall>
-        Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam
-      </Typography.TextSmall>
+    <Layout useMargins translationUrl={`/${useOtherLanguage()}/about`}>
```

Comment:
idk why i didnt think to use useOtherLanguage() like this, maybe because it is kinda like a hook? Are there warnings about using a hook in the console by doing this?

Reply:
i did make the `useOtherLanguage()` hook but I thought I always set it to a variable before the return? just a good practice for hooks since they can't be called conditionally, i.e. in the case where we do an early return or something